ports/gettext/portfile.cmake

```
@@ -41,25 +42,42 @@ if(VCPKG_TARGET_IS_WINDOWS)
                         # Left here for future reference. 
                         gl_cv_func_printf_directive_n=no #segfaults otherwise with popup window
                         ac_cv_func_memset=yes #not detected in release builds 
+                        ac_cv_header_pthread_h=no
                         )
 endif()
-vcpkg_configure_make(SOURCE_PATH ${SOURCE_PATH}/gettext-runtime # Port should probably be renamed to gettext-runtime instead of only gettext. Removing the subdir here builds all of gettext
-                     DETERMINE_BUILD_TRIPLET
-                     USE_WRAPPERS
-                     ADD_BIN_TO_PATH    # So configure can check for working iconv
-                     OPTIONS --enable-relocatable #symbol duplication with glib-init.c?
-                             --enable-c++
-                             --disable-java
-                             ${OPTIONS}
-                    )
-                    
+if("tools" IN_LIST FEATURES)
+    vcpkg_configure_make(SOURCE_PATH ${SOURCE_PATH} # Port should probably be renamed to gettext-runtime instead of only gettext. Removing the subdir here builds all of gettext
+                         DETERMINE_BUILD_TRIPLET
+                         USE_WRAPPERS
+                         ADD_BIN_TO_PATH    # So configure can check for working iconv
+                         OPTIONS --enable-relocatable #symbol duplication with glib-init.c?
+                                 --enable-c++
+                                 --disable-java
+                                 ${OPTIONS}
+                        ADDITIONAL_MSYS_PACKAGES gzip
+                        )
+else()
+    vcpkg_configure_make(SOURCE_PATH ${SOURCE_PATH}/gettext-runtime # Port should probably be renamed to gettext-runtime instead of only gettext. Removing the subdir here builds all of gettext
+                         DETERMINE_BUILD_TRIPLET
+                         USE_WRAPPERS
+                         ADD_BIN_TO_PATH    # So configure can check for working iconv
+                         OPTIONS --enable-relocatable #symbol duplication with glib-init.c?
+                                 --enable-c++
+                                 --disable-java
+                                 ${OPTIONS}
+                        )
+endif()
+
```

```suggestion
if("tools" IN_LIST FEATURES)
    set(ADDITIONAL_MSYS_PACKAGES gzip)
else()
    set(SOURCE_PATH ${SOURCE_PATH}/gettext-runtime)
endif()

vcpkg_configure_make(SOURCE_PATH ${SOURCE_PATH}
                     DETERMINE_BUILD_TRIPLET
                     USE_WRAPPERS
                     ADD_BIN_TO_PATH    # So configure can check for working iconv
                     OPTIONS --enable-relocatable #symbol duplication with glib-init.c?
                             --enable-c++
                             --disable-java
                             ${OPTIONS}
                     ADDITIONAL_MSYS_PACKAGES ${ADDITIONAL_MSYS_PACKAGES}
                    )

```
imho this is cleaner